components/ILIAS/Contact/classes/MailingLists/MailingListsTable.php

```diff
@@ -71,6 +71,7 @@ public function getComponent(): UI\Component\Table\Data
                 $columns,
                 $this
             )
+            ->withId(self::class . '_' . $this->mailing_lists->getCurrentMailingList()->getId())
```

Comment:
IMO this table does not need a suffix, because for every user there is only one list of the mailing lists.

See: `ilMailingListsTableGUI` in ILIAS 9.x